Breakout/src/game_clock.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <vector>
+#include <functional>
+#include <SFML/System/Clock.hpp>
+
+class GameClock
+{
+public:
+	/* Note: Clock starts at construction*/
+	explicit GameClock(float);
```

Comment:
Default parameters as well as the parameter name here for easier usage of the class and helping the C++ compiler understand which parameters are optional.
```suggestion
	explicit GameClock(float fps = 60.f);
```

Reply:
I've never quite known where the best place to put default params was. Thank you.